src/Styleguide/Pages/Artwork/index.js

```diff
@@ -5,6 +5,8 @@ import { AboutTheWork } from "./AboutTheWork"
 import { Bibliography } from "./Bibliography"
 import { ExhibitionHistory } from "./ExhibitionHistory"
 import { Banner } from "./Banner"
+import { Artists } from "./Sidebar/Artists"
+import { SingleFollowedArtist } from "./__stories__/Sidebar/Artists.story"
```

Comment:
Not sure what is the best way to show those different option here. For now one option will do.

Reply:
We could add a stateful click switch as we're developing. What i've done in the past for areas that have multiple options is to drop in a container that when clicked cycles through the different options.